app/lib/methods/subscriptions/rooms.js

```diff
@@ -407,7 +407,7 @@ export default function subscribeRooms() {
 	};
 
 	connectedListener = this.sdk.onStreamData('connected', handleConnection);
-	disconnectedListener = this.sdk.onStreamData('close', handleConnection);
+	// disconnectedListener = this.sdk.onStreamData('close', handleConnection);
```

Comment:
Remove

Reply:
We've been changing this a lot. I want to leave it there for now.